### PR TITLE
fix #3213: remove P tags from submissions on private leaderboards

### LINKED ISSUE #3213: [Frontend_v2] P tag should not appear for private leaderboards
### Description

Screenshot_2020-10-11_at_11 38 42_AM

P tag should not appear for submissions on the private leaderboards. And change column `Last Submitted At` to `Submitted At` on the leaderboard

### PR DESCRIPTION
**Pull request by burnerlee**
resolves #3213

### CODE REVIEW

**Comment by burnerlee** [comment]
Screenshot from 2021-03-09 17-17-00
No P tag is shown for the private submission made because the leaderboard for John Phase is not public

File: frontend_v2/src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.ts [hunk 1]
```diff
@@ -110,6 +110,11 @@ export class ChallengeleaderboardComponent implements OnInit, AfterViewInit, OnD
    */
   showLeaderboardUpdate = false;
 
+  /**
+   * Returns true is leaderboard is public for the selected phase
```
Comment (Ram81) [inline frontend_v2/src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.ts:114]: Change `Returns true is leaderboard is public for the selected phase` to `Returns true if leaderboard is public for the selected phase`

File: frontend_v2/src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.ts [hunk 2]
```diff
@@ -110,6 +110,11 @@ export class ChallengeleaderboardComponent implements OnInit, AfterViewInit, OnD
...
+   */
+  isLeaderboardPublic = true;
```
Comment (Ram81) [inline frontend_v2/src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.ts:116]: rename to `isSelectedPhaseLeaderboardPublic`

File: frontend_v2/src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.ts [hunk 3]
```diff
@@ -619,4 +628,5 @@ export class ChallengeleaderboardComponent implements OnInit, AfterViewInit, OnD
   ngOnDestroy() {
     clearInterval(this.pollingInterval);
   }
+
```
Comment (Ram81) [inline frontend_v2/src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.ts:631]: remove newline

**Review by Ram81 — CHANGES_REQUESTED** [review]
@burnerlee can you please make these changes

**Comment by burnerlee** [comment]
@Ram81 made the fixes

**Review by Ram81 — APPROVED** [review]
LGTM 👍

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**